docs/sphinx/source/whatsnew/v0.8.0.rst

```diff
@@ -14,6 +14,8 @@ Bug fixes
 
 Testing
 ~~~~~~~
+* Decorator :py:func:`pvlib.conftest.fail_on_pvlib_version` now accepts
+  arguments that are passed to the decorated function. (:pull:`973`)
```

Comment:
This isn't true anymore.  Maybe something like this?

```
* Decorator :py:func:`pvlib.conftest.fail_on_pvlib_version` can now
  be used to decorate functions that take arguments. (:pull:`973`)
```

Reply:
Yes, sorry my oversight here and for not removing `*args, **kwargs`